certified-connectors/Yarado/apiDefinition.swagger.json

```diff
@@ -0,0 +1,165 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Yarado",
+    "description": "Yarado connector lets users list down robots and tasks belonging to their organisation and schedule a single task run.",
+	"contact": {
+	  "name": "Yarado Customer Service Center",
+	  "url": "https://yarado.atlassian.net/servicedesk/customer/portals",
+	  "email": "info@yarado.com"
+	}  
+  },
+  "host": "api.yarado.com",
+  "basePath": "/",
+  "x-ms-connector-metadata": [
+    {
+	  "propertyName": "Website",
+	  "propertyValue": "https://yarado.com"
+	},
+	{
+	  "propertyName": "Privacy policy",
+	  "propertyValue": "https://www.yarado.com/terms"
+	}, 
+	{
+	  "propertyName": "Categories",
+	  "propertyValue": "Business Management;Productivity"
+	}
+  ],
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/robots": {
+      "get": {
+        "summary": "GET Robots",
+        "description": "GET Robots",
+        "operationId": "GetRobots",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string"
+                  },
+                  "name": {
+                    "type": "string"
+                  },
+                  "organisation_name": {
+                    "type": "string"
+                  },
+                  "is_deleted": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/task-files": {
+      "get": {
+        "summary": "GET Task Files",
+        "description": "GET Task Files",
+        "operationId": "GetTaskFiles",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                  "type": "string"
+                  },
+                  "filename": {
+                    "type": "string"
+                  },
+                  "created_by": {
+                    "type": "string"
+                  },
+                  "created_at": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v1/task-schedules": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@MarcinKowalczukYarado empty schemas are not allowed. Please provide valid responses for your operations. You can refer to [instructions from the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses) and [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).